### DISCUSSION TITLE
slight logo changes..

### DISCUSSION THREAD

**Discussion by rfdixon** [eclipse-theia/theia#14129] [Ideas]
Not sure where to add this. I was playing around with this for my own environment and thought I'd offer it up here.
I added the orange "bow shock" to tie it to eclipse family and shortened it so it didn't feel too proportionally different from other icons in my task bar.
![image](https://github.com/user-attachments/assets/7e5f795a-b9e0-412c-a831-0cf0a7940248)

![image](https://github.com/user-attachments/assets/c82d3c5b-7cb3-4c7a-8aee-853ac098c438)

**Comment by JonasHelming** [comment]
Thank you for sharing this, I will add this to the [dev call agenda](https://github.com/eclipse-theia/theia/wiki/Dev-Meetings)

**Comment by JonasHelming** [comment]
People liked it a lot. Would you be able to contribute this formally, including an SVG? Ideally here: https://github.com/eclipse-theia/theia-blueprint/tree/master/TheiaIDE%20logo

**Reply by rfdixon** [reply]
sure thing! I'll go ahead and put together the entire logo as well. Should I post it here for approval before submitting?

**Reply by JonasHelming** [reply]
As you like. Reviewing here is probably a ore lighweight process
